client/src/model/store.ts

```diff
@@ -1,10 +1,29 @@
-import { configureStore } from '@reduxjs/toolkit';
-import geosReducer from '@/model/geo_slice';
+import { inject, computed } from 'vue';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import geosReducer from '@/model/slices/geo_slice';
+import leadDataReducer from '@/model/slices/lead_data_slice';
+import { storeKey } from '@/model/store_plugin';
+
+const rootReducer = combineReducers({
+  geos: geosReducer,
+  leadData: leadDataReducer,
+});
+
+const store = configureStore({
+  reducer: rootReducer,
+});
 
-const store = configureStore({ reducer: { geosReducer } });
 // Emits events for reducers.
 export const dispatch = store.dispatch;
 
+// Same as https://react-redux.js.org/api/hooks#useselector
+export const useSelector = <State extends RootState = RootState>(
```

Comment:
Can you add to this comment 'Allows you to extract data from the Redux store state.'

Reply:
Yep!